src/components/featured-article/macro.spec.js

```diff
@@ -60,11 +60,32 @@ describe('Macro: Featured Article', () => {
                 expect($('.ons-image__img').attr('alt')).toBe('Example alt text');
             });
         });
+
+        describe('WHEN: both chart and image is provided', () => {
+            const $ = cheerio.load(renderComponent('featured-article', EXAMPLE_FEATURED_ARTICLE_WITH_CHART));
+
+            test('THEN: it renders the onsChart component', () => {
+                expect($('[data-highcharts-base-chart]').length).toBe(1);
+            });
```

Comment:
Maybe we also want to test that it doesn't render an image

Reply:
done